doc/lsst.pipe.tasks/index.rst

```diff
@@ -9,7 +9,7 @@ The :ref:`command-line tasks <lsst.pipe.tasks-command-line-tasks>` listed here a
 You can also assemble your own pipelines by combining individual tasks through their Python APIs.
 
 ``lsst.pipe.tasks`` does not provide all the tasks and command-line tasks in the LSST Science Pipelines.
-For a complete overview of the available tasks, see the *Processing Data* documentation section (to be completed).
+For a complete overview of the available tasks, see `Getting started with the LSST Science Pipelines <https://pipelines.lsst.io/getting-started/index.html>`_.
```

Comment:
A couple things. First, it's important to use a `doc` or `ref` role to make this link so that it's a _relative_ link within a documentation edition.

```suggestion
For an introduction to processing data, see :doc:`/getting-started/index`.
```

Second, the introductory sentence before this doesn't correspond to the change you made. This second sentence was meant to link to an overview of all the processing tasks. We still don't have that section in the docs yet, but for now a better link would be

```suggestion
For an complete list of all available tasks, see :doc:`/tasks`
```

(which links to https://github.com/lsst/pipelines_lsst_io/blob/master/tasks.rst)

So what's I'd recommend is that you use the second suggestion, but add the first suggestion to either a new paragraph, or couple it to the start of the paragraph that follows this one.

Reply:
Sure, no problem. Change made. Someone had suggested that the link to the *Processing Data* documentation was meant to point to the getting started guide but had not been updated, which is why I made the change the way that I did.